### PR TITLE
[playstation] Clarified the part to obtain configuration info

### PR DESCRIPTION
**Pull request by Balisz**
Hello,
I am a complete newbie at this and it was really difficult to understand how to obtain the user credential with the provided instructions.
I hope this additional info will make it easier for others.

### CODE REVIEW

File: bundles/org.openhab.binding.playstation/README.md [hunk 1]
```diff
@@ -30,8 +30,7 @@ Settings -> System Settings -> Connect PS Vita System Using Network.
...
 | ipPort              |  997    | No       | The port to probe the PS4 on, no need to change normally.                |
 
-If you want to control your PS4 the first thing you need is your user-credentials, this is a 64 characters HEX string that is easiest obtained by using PS4-waker https://github.com/dhleong/ps4-waker.
-The result file is called ".ps4-wake.credentials.json" in your home directory.
+If you want to control your PS4 the first thing you need is your user-credentials, this is a 64 characters HEX string that is easiest obtained by using PS4-waker https://github.com/dhleong/ps4-waker. To run the PS4-waker you will need a Node.js command prompt (for example: https://nodejs.org/en/download/). Enter "npx ps4-waker --help" int the command prompt. Agree to install by entering "y". After that send "npx ps4-waker --check". You will get asked to connect the "PS4 Second screen" android app to the running clone. Do this and then you will need to get the pairing key from your PS4 --> Settings ---> Mobile device pairing settings. On the PS4 screen you will see your pairing code and in the command prompt you will find the user credential.
```
Comment (jlaur) [inline bundles/org.openhab.binding.playstation/README.md:33]: typo: "in**t** the command prompt"

Comment (jlaur) [inline bundles/org.openhab.binding.playstation/README.md:33]: "you will find the user credential" - user credential**s**?

Comment (jlaur) [inline bundles/org.openhab.binding.playstation/README.md:33]: "**A**ndroid app" with capital A?

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! A few minor comments.

File: bundles/org.openhab.binding.playstation/README.md (hunk 1 above)
Comment (Balisz) [reply bundles/org.openhab.binding.playstation/README.md]: yes

Comment (Balisz) [reply bundles/org.openhab.binding.playstation/README.md]: yes

Comment (Balisz) [inline bundles/org.openhab.binding.playstation/README.md]: Done

**Review by Balisz — COMMENTED** [review]
Resolved requested actions

**Review by jlaur — APPROVED** [review]
Thanks for this contribution to the documentation.

**Pull request merged**